### PR TITLE
Allow small test genesis file to be used in mocknet mode

### PR DESCRIPTION
**Pull request by zone117x**
Closes https://github.com/blockstack/stacks-blockchain/issues/2176

### CODE REVIEW

File: testnet/stacks-node/src/node.rs [hunk 1]
```diff
@@ -181,6 +182,17 @@ fn spawn_peer(
 impl Node {
     /// Instantiate and initialize a new node, given a config
     pub fn new(config: Config, boot_block_exec: Box<dyn FnOnce(&mut ClarityTx) -> ()>) -> Self {
+        let use_test_genesis_data = if config.burnchain.mode == "mocknet" {
+            // When running in mocknet mode allow the small test genesis chainstate data to be enabled.
+            if env::var("BLOCKSTACK_USE_TEST_GENESIS_CHAINSTATE") == Ok("1".to_string()) {
```
Comment (jcnelson) [inline testnet/stacks-node/src/node.rs:187]: Instead of making this an envar, could we add this as a config option?

Comment (zone117x) [reply testnet/stacks-node/src/node.rs:187]: We could, although env vars are easier to deal with in the tooling than config files. How strongly do you prefer config file vs env var?

Comment (friedger) [reply testnet/stacks-node/src/node.rs:187]: It would be nice for the competition to set this as a config option

Comment (zone117x) [reply testnet/stacks-node/src/node.rs:187]: Are we okay with allowing both methods? Config or env var

Comment (zone117x) [reply testnet/stacks-node/src/node.rs:188]: @jcnelson @friedger Please see latest commit where support for both config toml and env var is added.

Comment (friedger) [reply testnet/stacks-node/src/node.rs:188]: Great

**Review by kantai — APPROVED** [review]
LGTM!

File: testnet/stacks-node/src/node.rs (hunk 1 above)
Comment (jcnelson) [reply testnet/stacks-node/src/node.rs:188]: Both are fine.

**Review by jcnelson — APPROVED** [review]
LGTM! Thanks @zone117x

**Pull request merged**